Configuration/Settings.yaml

```diff
@@ -14,3 +14,7 @@ Neos:
 
       restrictByOldUriPrefix: []
       #  '/some/uri/path/': true
+
+      # in some cases you might need to completely disable the automatic redirects
+      # e.g. on cli, during imports or similar
+      disableAutomaticRedirects: false
```

Comment:
We already have `enableRemovedNodeRedirect`. Maybe we should use the same naming scheme and set it to enabled by default?

Reply:
I wanted to explicitly disable instead of having it implicitly enabled, but either way is ok. I'll push it in a minute